e2e-test/src/test/features/addfield/RunTimeScenarioAddField.feature

```diff
@@ -0,0 +1,50 @@
+@Send_Email
+Feature: Send Email Alert - Run time scenarios
+
+  @BQ_SINK_TEST @FILE_SOURCE_TEST
+  Scenario: Verify add field functionality using File to BigQuery pipeline is successful
```

Comment:
Verify add field plugin functionality by setting field value using File to BigQuery pipeline